### PR TITLE
Add more metrics to the services

### PR DESCRIPTION
**Pull request by andriikushch**
This PR:
- Adds more metrics to the services
- Adds new services
- Adds dimensions to the new services

### CODE REVIEW

**Review by kgeckhart — APPROVED** [review]
:shipit:

File: pkg/cloudWatchConsts/metrics.go [hunk 1]
```diff
@@ -3083,4 +3204,6 @@ var NamespaceDimensionKeysMap = map[string][]string{
 	"AWS/Cassandra":                               {"Keyspace", "Operation", "TableName"},
 	"AWS/AmplifyHosting":                          {"App"},
 	"CloudWatchSynthetics":                        {"CanaryName", "StepName"},
```
Comment (iwysiu) [inline pkg/cloudWatchConsts/metrics.go:3206]: Even if it's empty we should add `AWS/ECR` here (I think it errors if it doesn't exist)

**Review by iwysiu — APPROVED** [review]
Looks good with one change!

File: pkg/cloudWatchConsts/metrics.go (hunk 1 above)
Comment (andriikushch) [reply pkg/cloudWatchConsts/metrics.go:3206]: Thank you @iwysiu!

Added here https://github.com/grafana/grafana-aws-sdk/pull/161/commits/692a75fa2e53a0b0c7aa422ae16d680972403aba.

**Pull request merged**